src/main/java/com/entropyteam/entropay/employees/models/Gender.java

```diff
@@ -0,0 +1,7 @@
+package com.entropyteam.entropay.employees.models;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+
+@JsonFormat(shape = JsonFormat.Shape.STRING)
+public enum Gender{Male, Female, NO_BINARY}
```

Comment:
please change to MALE and FEMALE to keep the format convention

Reply:
Changed